iOS/Flows/Conversation/Cells/ConversationMessageCellDatasource.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  ConversationMessageCellDatasource.swift
+//  Jibber
+//
+//  Created by Martin Young on 11/3/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+typealias ConversationMessageSection = ConversationMessageCellDataSource.SectionType
+typealias ConversationMessageItem = ConversationMessageCellDataSource.ItemType
+
+class ConversationMessageCellDataSource: CollectionViewDataSource<ConversationMessageSection,
+                                            ConversationMessageItem> {
```

Comment:
Here's the data source. I basically transplanted the datasource code from the message cell into this.